compare_mt_assembly_to_ref.md

```diff
@@ -0,0 +1,327 @@
+# Comparing our de novo transcriptome assembly to the reference
+
+In the [transcriptome assembly lesson](https://angus.readthedocs.io/en/2018/transcriptome-assembly.html), we assembled short RNA reads into contigs. Here we will compare this to the reference transcriptome we worked with in the [mapping and variant calling](https://angus.readthedocs.io/en/2018/mapping-variant-calling.html) lesson, and get some more practice at the command line and working with blast!  
+
+---
+
+Learning objectives:
+
+	* Explore some more bash commands in a practical way (e.g. `grep`, `sed`, `cut`, `comm`, `cat`)
+	* Practice stringing together multiple commands with pipes (`|`) step-by-step
+	* Use blast to compare two transcriptomes to find sequences that are different
+	* Run a "remote" blast from the command line
+
+## Boot up a Jetstream
+[Boot an m1.medium Jetstream instance](https://angus.readthedocs.io/en/2018/jetstream/boot.html) and log in.
+
+## Software needed
+We will be using blast here, so if you've run through the [using blast at the command line](https://angus.readthedocs.io/en/2018/running-command-line-blast.html) lesson, you're already good to go. You can check that you have blast by entering:
+
+```bash
+makeblastdb -h
+```
+
+If that returns some help information, blast is already installed. If it returns a message with "command not found", then you can install blast with:
+
+```bash
+conda install -y blast
+```
+
+## Setting up our working directory
+
+Let's set up a new working directory to compare our assembly to the orf reference and copy over the reference we worked with and the result from our de novo transcriptome assembly (they're both small enough files that copying isn't a big deal).
+
+> NOTE: ".fa", ".fasta", and ".fna" are all common extensions for a nucleotide fasta file. Amino acid fasta files typically have ".faa".
+
+```bash
+cd ~
+mkdir comparing_assembly_to_ref
+cd comparing_assembly_to_ref/
+cp ../mapping/orf_coding.fasta .
+cp ../assembly/yeast-transcriptome-assembly.fa .
+```
+
+> If either of those returned an error message (file not found), you can download the reference by running `curl -O https://downloads.yeastgenome.org/sequence/S288C_reference/orf_dna/orf_coding.fasta.gz` followed by `gunzip orf_coding.fasta.gz` to download and unzip the reference, and/or `curl -O https://ndownloader.figshare.com/files/12323351` to download a result of the transcriptome assembly (this one is not gzipped).
```

Comment:
the last curl command doesn't name the file, does it?

Reply:
oh, oops, no it doens't, only flag should be `-L`, thanks!